ListingService/Sources/CoreAPI+Listings.swift

```diff
@@ -16,4 +16,17 @@ extension CoreAPI {
             }
         }
     }
+    static func newListings(afterListing: Listing? = nil,
```

Comment:
`newsListings` instead of `newListings`

Reply:
Fixed in 
 https://github.com/kgellci/Area51/pull/12/commits/418491b125b2c087d6123c7771f732eba69235c3